pkg/apis/admission/types.go

```diff
@@ -63,7 +63,9 @@ type AdmissionRequest struct {
 	// Namespace is the namespace associated with the request (if any).
 	// +optional
 	Namespace string
-	// Operation is the operation being performed
+	// Operation is the operation being performed. This may be different than the operation
+	// requested. Both a patch and create request can result in either a CREATE or UPDATE
+	// Operation.
```

Comment:
is this accurate? how does a create request turn into an update? I know update can turn into create in some cases (if AllowCreateOnUpdate is true)

Reply:
Even for the CreateOnUpdate case, can you double check if the apiserver sends CREATE or UPDATE in the admission review request?

Reply:
I think I got this backwards. I'll just say "e.g. a patch can result in either a CREATE or UPDATE Operation."

> Even for the CreateOnUpdate case, can you double check if the apiserver sends CREATE or UPDATE in the admission review request?

Here: https://github.com/kubernetes/kubernetes/blob/0252a323423080a3a230ab3053fdaaca8a972eb1/staging/src/k8s.io/apiserver/pkg/endpoints/handlers/update.go#L141

Reply:
yeah, @jennybuckley fixed up all of those cases when wiring up apply -> {create,update} paths

Reply:
> Both a patch and create request can result in either a CREATE or UPDATE Operation.

comment on this bit is still outstanding, right?

Reply:
It's applied now.